packages/tools/package.json

```diff
@@ -49,7 +49,7 @@
         "@inquirer/select": "2.3.3",
         "@rollup/plugin-commonjs": "28.0.1",
         "@rollup/plugin-node-resolve": "15.3.0",
-        "@rspack/core": "1.1.2",
+        "@rspack/core": "1.3.10",
```

Comment:
Shouldn't this be a peer dependency btw?

Same for esbuild & waebpack?

Reply:
Related, if this is for internal, shouldn't we bump the requirement on rspack to at least the 1.3.10 in the peerDeps (as it'll fail for older versions)

Reply:
It's internal only here.

I thought about bumping the requirements in the public ones, but I feel like it's a big restriction and higher the bar of entry for the plugin.
The consequences are not breaking anything, it's just when using a specific hook with rspack (`shutdown`) it "sometimes" fail (once every 20 or so builds).
It felt low risk to really justify such bump in requirement.
I'll revise if I get reports over this though.

Reply:
Noted, thanks!